nussl/audio_signal.py

```diff
@@ -1170,11 +1233,32 @@ def to_mono(self, overwrite=False):
             (:obj:`np.array`): Mono-ed version of :attr:`audio_data`.
 
         """
-        mono = np.mean(self.audio_data, axis=constants.CHAN_INDEX)
+        if remove_channels:
+            mono = np.mean(self.audio_data, axis=constants.CHAN_INDEX)
+        else:
+            mono = np.mean(self.audio_data, axis=constants.CHAN_INDEX, keepdims=True)
         if overwrite:
             self.audio_data = mono
         return mono
 
+    def stft_to_mono(self, overwrite=False):
+        """ Converts :attr:`stft_data` to mono by averaging every sample.
```

Comment:
Hmm... maybe specify that it will be one channel. I'm used to seeing the term 'mono' used in the time domain, but not in the time-frequency domain. Also, what will the shape of `stft_data` be after you run this function (Hint: `[NUM_FREQ, NUM_HOPS, 1]`)?